src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ListingDetailsSteps.java

```diff
@@ -266,14 +266,16 @@ public void iLookAtG1G2Measures() {
     /**
      * Asserts Measure required text and associated criteria show correct.
      * @param mName expected text
-     * @param reqText is expected toggle tip text
+     * @param reqTest is expected toggle tip text
      * @param criteria is criteria number
      */
-    @Then("^the \"([^\"]*)\" shows Required Text as \"([^\"]*)\" and associated criteria as \"([^\"]*)\"$")
-    public void assetRequiredText(final String mName, final String reqText, final String criteria) {
-        String actualString = ListingDetailsPage.requiredText(driver, mName).getText();
-        assertTrue(actualString.contains(reqText), "Expect \"" + reqText + "\" to be found in \"" + actualString + "\"");
+    @Then("^the Required Test shows as \"([^\"]*)\" for \"([^\"]*)\" and associated criteria \"([^\"]*)\"$")
+    public void requiredTestValidationBasedOnMeasureName(final String reqTest, final String mName, final String criteria) {
+        String actualString = ListingDetailsPage.measureNameForReqTest(driver, reqTest).getText();
+        System.out.println(actualString);
+        assertTrue(actualString.contains(mName), "Expect \"" + mName + "\" to be found in \"" + actualString + "\"");
         String associatedCriteria = ListingDetailsPage.associatedCriteria(driver, mName).getText();
+        System.out.println(associatedCriteria);
```

Comment:
Looks like a debugging line. Probably not needed any more, right?

Reply:
yes, forgot to remove it. I took it out, change will be in next PR